ucsb-cs156-f24/proj-happycows-f24-11
# Add Put Endpoint for Single Student

## Overview

In this PR, I add a put endpoint for a single student in the backend.

## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go on swagger and check put endpoint.

## Dokku Link
https://happycows-blobthebuilder.dokku-11.cs.ucsb.edu
This dokku link is being used for pull #54 right now.
 
## Tests


## Linked Issues 
Closes #36
